Sources/GRPCHTTP2TransportNIOPosix/HTTP2ServerTransport+Posix.swift

```diff
@@ -174,6 +178,15 @@ extension HTTP2ServerTransport {
         }
       }
 
+      #if canImport(NIOSSL)
+      let nioSSLContext: NIOSSLContext?
+      if let tlsConfig = self.config.transportSecurity.tlsConfig {
+        nioSSLContext = try! NIOSSLContext(configuration: TLSConfiguration(tlsConfig))
```

Comment:
Oops, this was an unintended leftover from when I was first putting things together.
I can just leave the `try`: this is `listen`, so the server won't even start up if this throws (meaning it's not that we need an `RPCError` here to send back to the client). Whatever error we wrap this in won't be able to add any other helpful information than the actual root cause. 
Not sure what the value is in wrapping it in this case?

Reply:
I think there is some value: we can provide an obvious high level failure message: "Invalid TLS configuration" and then attach the underlying error to that. It might be worth explicitly catching some of the NIOSSL errors and providing better error messages too.